.github/workflows/build_fluid_docs.yml

```diff
@@ -27,7 +27,15 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Install prerequisites
-      run: sudo apt-get update -y && sudo apt-get install -y libwayland-dev wayland-protocols libdbus-1-dev libxkbcommon-dev libpango1.0-dev libgtk-3-dev
+      run: |
+        sudo apt-get update -y
+        sudo apt-get install -y libwayland-dev wayland-protocols libdbus-1-dev libxkbcommon-dev libpango1.0-dev libgtk-3-dev
+        sudo apt-get install -y doxygen=1.10.0
+        sudo apt-get install texlive-latex-base
+        sudo apt-get install texlive-fonts-recommended
+        sudo apt-get install texlive-latex-extra
```

Comment:
@MatthiasWM Instead of all these `texlive-*` packages installing `doxygen-latex` should do it. This should also imply `doxygen` but I don't know if `doxygen=1.10.0` works (?)

Reply:
Nope, Ubuntu is at 1.9.2. 1.10.0 does not work :-(